protocol-spec/SUMMARY.md

```diff
@@ -42,11 +42,19 @@
   * [StopAllDevices](generic.md#stopalldevices)
   * [RawCmd](generic.md#rawcmd)
   * [SingleMotorVibrateCmd](generic.md#singlemotorvibratecmd)
+  * [VibrateCmd](generic.md#vibratecmd)
+  * [LinearCmd](generic.md#linearcmd)
+  * [RotateCmd](generic.md#rotatecmd)
+  * [ConstrictCmd](generic.md#constrictcmd)
 * [Specific Device Messages](specific.md#specific-device-messages)
   * [KiirooCmd](specific.md#kiiroocmd)
   * [FleshlightLaunchFW12Cmd](specific.md#fleshlightlaunchfw12cmd)
   * [LovenseCmd](specific.md#lovensecmd)
   * [VorzeA10CycloneCmd](specific.md#vorzea10cyclonecmd)
+* [Generic Sensor Messages](input.md#generic-sensor-messages)  
+  * [StartAccelerometerCmd](input.md#startaccelerometercmd)
```

Comment:
I'd really like to leave all accelerometer stuff to > 0.2. I don't feel like we have a good enough handle of the use cases yet, and 0.2 is already packed as is.